site/homepage/content/community/thankyou.md

```diff
@@ -0,0 +1,14 @@
++++
+linkTitle = "Thank you"
+title = "Organizations supporting the Development of Hono"
+weight = 650
+[menu.main]
+    parent = "Community"
```

Comment:
As the `Thank  you` page is now moved under `community`, the reference to the `Thank you` link in the _footer_  should be updated accordingly.

https://github.com/eclipse/hono/blob/91949f4d23ad78a5eb181086a9cdfc6bf9f5f85e/site/homepage/layouts/partials/footer.html#L11